addons/website_crm_partner_assign/controllers/main.py

```diff
@@ -250,13 +265,8 @@ def partners(self, country=None, grade=None, page=0, **post):
             'active': bool(grade is None),
         })
 
-        # group by country
-        country_domain = list(base_partner_domain)
-        if grade:
-            country_domain += [('grade_id', '=', grade.id)]
-        countries = partner_obj.sudo().read_group(
-            country_domain, ["id", "country_id"],
-            groupby="country_id", orderby="country_id")
+
```

Comment:
too empty lines spaces (and also one line too little before `@patch('odoo.http.GeoIP')`